### PR TITLE
Creating separate component for evaluators dialog

### PR DESCRIPTION
**Pull request by MingoLi**
[US135227](https://rally1.rallydev.com/#/?detail=/userstory/622349617123&fdp=true): FE: Creating separate component for evaluators dialog

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor.js [hunk 1]
```diff
@@ -1,74 +1,29 @@
...
-import '@brightspace-ui/core/components/inputs/input-search.js';
-import '@brightspace-ui/core/components/list/list.js';
-import '@brightspace-ui/core/components/list/list-item.js';
-import '@brightspace-ui/core/components/list/list-item-content.js';
-import '@brightspace-ui/core/components/loading-spinner/loading-spinner.js';
-import '@brightspace-ui/core/components/selection/selection-action.js';
-import 'd2l-users/components/d2l-profile-image.js';
+import './d2l-activity-evaluators-editor-dialog.js';
 import { css, html } from 'lit-element/lit-element.js';
 import { sharedSelectedEvaluators, sharedEvaluators as store,  } from './state/activity-evaluation-store.js';
```
Comment (alexvlsem) [inline components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor.js:3]: What do you think if we remove sharedEvaluators (it is not used here anymore) and use sharedSelectedEvaluators as store?

Comment (MingoLi) [reply components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor.js:3]: In this case, are `sharedSelectedEvaluators.isEnabled()` and `sharedEvaluators .isEnabled()` doing the same check?

Comment (alexvlsem) [reply components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor.js:3]: Yes. Anything that uses sharedEvaluators  can be removed as well.

File: components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor-dialog.js [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+import { accordionStyles } from '../styles/accordion-styles';
+import { ActivityEditorMixin } from '../mixins/d2l-activity-editor-mixin.js';
+import { asyncStates } from '@brightspace-ui/core/mixins/async-container/async-container-mixin.js';
+import { labelStyles } from '@brightspace-ui/core/components/typography/styles.js';
+import { LocalizeEvaluationMixin } from './mixins/d2l-activity-evaluation-lang-mixin.js';
+import { MobxLitElement } from '@adobe/lit-mobx';
+import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
+
+const
+	dialogId = 'activity-evaluators-dialog-editor',
```
Comment (alexvlsem) [inline components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor-dialog.js:23]: Shouldn't we remove dialogId?

Comment (MingoLi) [reply components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor-dialog.js:23]: yeah right, will do

File: components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor-dialog.js [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+	async _loadMore() {
+
+		this._isLoadMoreRunning = true;
+
+		await this._state.loadMore();
+
+		this._isLoadMoreRunning = false;
+	}
+
+	async _open() {
```
Comment (alexvlsem) [inline components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor-dialog.js:170]: This method is used in d2l-activity-evaluators-editor. Shouldn't it be a public one?

Comment (MingoLi) [reply components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluators-editor-dialog.js]: Yes!

**Review by alexvlsem — APPROVED** [review]

**Pull request merged**